tests/testMatchOptimisticB.py

```diff
@@ -132,8 +133,9 @@ def computePosRefCatalog(self, sourceCat):
         refCat = afwTable.SimpleCatalog(minimalPosRefSchema)
         for source in sourceCat:
             refObj = refCat.addNew()
-            refObj.set("coord", source.get("coord"))
-            refObj.set("centroid", source.getCentroid())
+            refObj.setCoord(source.getCoord())
+            refObj.set("centroid_x", source.getX())
+            refObj.set("centroid_y", source.getY())
```

Comment:
Point taken. I will note that tests are often used as examples, and using keys is a better example. But as you say...it's a lot of extra code for no performance gain.